views/about.erb

```diff
@@ -0,0 +1,60 @@
+<html>
+  <head>
+    <title>"Meighs' Page"</title>
+    <meta name="viewport" content="width=device width">
+  </head>
+  <body>
+      <header><center>
+        <h1>About Me</h1>
+        <p>Here's some stuff you should know about me...</p>
+        <img src = "./image1.jpg" width = "200" height = "150">
```

Comment:
In Ruby we like to add spaces between our `=` to make it easier to read, but HTML standards encourage us to not put those spaces before and after the `=`